package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ol",
-  "version": "6.5.1-dev",
+  "version": "6.5.1-module.1",
```

Comment:
I'll change this back if we decide to merge.  I've just done this here to test out `ol@module` tagged packages.